newsrevenuehub/rev-engine
# DEV-2719: Include "Organization" link in avatar menu and link to Org Settings Page

Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)
This is the second PR of stacked PRs:

1. Org settings page: https://github.com/newsrevenuehub/rev-engine/pull/994
2. Link into AvatarMenu (this PR)
3. Enhancement (Warn message) - https://github.com/newsrevenuehub/rev-engine/pull/996
4. Handle Fiscal Sponsor in Org Settings - https://github.com/newsrevenuehub/rev-engine/pull/997

#### What's this PR do?
- Adds a new Material UI icon library that is more complete than the other we have and the long-term objective is to remove the older lib and swap all icons to this new lib. Talked about this with @nrh-cklimas.
- Update TS, jest & package.json to work with svgr icons
- Added Organization menu item to AvatarMenu
- Create a new protected router wrapper to handle AC 4. `users with access to multiple orgs, and users with access to multiple orgs cannot navigate to these pages`

#### Why are we doing this? How does it help us?
- Make it possible for users to navigate to Organization page
- Standardize icon library

#### How should this be manually tested? Please include detailed step-by-step instructions.
Test all 3 scenarios:
1) - Log in with HUB admin
    - Check that AvatarMenu does not have Organization button
    - Access "settings/organization" in the URL and make sure it works

2) - Log in with user with access to multiple Orgs
    - Check that AvatarMenu does not have Organization button
    - Access "settings/organization" in the URL and make sure it redirects to "/pages"

3) - Log in with user with access to 1 Org
    - Check that AvatarMenu has Organization button and that it works (redirects to "settings/organization")
    - Access "settings/organization" in the URL and make sure it works

#### Did this PR make changes to the user interface that may require changes to the user-facing docs?
yes

#### Have automated unit tests been added? If not, why?
yes

#### How should this change be communicated to end users?
Users now have access to settings/organization page

#### Are there any smells or added technical debt to note?
no

#### Has this been documented? If so, where?
no

#### What are the relevant tickets? Add a link to any relevant ones.
https://news-revenue-hub.atlassian.net/browse/DEV-2719

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?
no

#### Are there next steps? If so, what? Have tickets been opened for them? List next-step tickets here:
no
